nyc_taxis/README.md

```diff
@@ -73,6 +73,7 @@ This workload allows to overwrite the following parameters using `--workload-par
 * `error_level` (default: "non-fatal"): Available for bulk operations only to specify ignore-response-error-level.
 * `target_throughput` (default: default values for each operation): Number of requests per second, `none` for no limit.
 * `search_clients`: Number of clients that issues search requests.
+* `trip_distance_type` (default: { "scaling_factor": 100, "type": "scaled_float" }): The `trip_distance` field type mapping
```

Comment:
This does not seem right.  The literal `trip_distance_type` refers not just to a type, but also includes the scaling factor, so it should probably be renamed.

Reply:
> `trip_distance_type`

Sure, like `trip_distance_field_mapping` or `trip_distance_mapping`?

Reply:
The former seems more appropriate.

Reply:
So `trip_distance_mapping` it is, any more comments? thank you!